ansible/roles/debops.docker_server/templates/etc/ansible/facts.d/docker.fact.j2

```diff
@@ -9,7 +9,7 @@ import subprocess
 import os
```

Comment:
You should rename the fact file to the `docker_server.fact.j2` as well, don't forget to update the task that templates it.